src/object/obj_internal.h

```diff
@@ -115,6 +115,12 @@ struct dc_object {
 	unsigned int		 cob_version;
 	unsigned int		 cob_shards_nr;
 	unsigned int		 cob_grp_size;
+	unsigned int		 cob_grp_count;
```

Comment:
minor, "cob_grp_nr" maybe more align with naming with other places

Reply:
Right, I will fix it.